javascript/src/test/pages/Home/Home.test.js

```diff
@@ -1,28 +1,101 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import Home from "main/pages/Home/Home";
+import { Jumbotron } from "react-bootstrap";
+import { waitFor, render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
```

Comment:
First and third imports aren't being used, `waitFor` is not being used in second import